### PR TITLE
Update Using Sonatype Page

### PR DESCRIPTION
**Pull request by Tirithel**
## Description of change:
- Updates `sbt-sonatype` version to `v3.9.13` to support accounts created after Febuary 2021.
- Modified all references of `oss.sonatype.org` to also comment about using `s01.oss.sonatype.org` for newer accounts.
- Ran scalafmt on the file, removed a lot of whitespace. Should we add scalafmt as a buildstep?

**_Added comments to relevant changes, everything else is whitespace._**

### CODE REVIEW

File: src/reference/01-General-Info/05-Using-Sonatype.md [hunk 1]
```diff
@@ -139,7 +139,9 @@ password=<your password>
 ```
 
 > *Note:* The first two strings must be `"Sonatype Nexus Repository Manager"`
-and `"oss.sonatype.org"` for Coursier to use the credentials.
+> and `"oss.sonatype.org"` for Coursier to use the credentials. If you are using
+> a new OSSRH account created after February 2021, use `"s01.oss.sonatype.org"`
+> instead of `"oss.sonatype.org"`
```
Comment (Tirithel) [inline src/reference/01-General-Info/05-Using-Sonatype.md:144]: Adding information about s01 change for accounts made after Feb 2021.

File: src/reference/01-General-Info/05-Using-Sonatype.md [hunk 2]
```diff
@@ -161,28 +163,32 @@ ThisBuild / scmInfo := Some(
...
 
 // Remove all additional repository other than Maven Central from POM
 ThisBuild / pomIncludeRepository := { _ => false }
 ThisBuild / publishTo := {
+  // For accounts created after Feb 2021:
+  // val nexus = "https://s01.oss.sonatype.org/"
   val nexus = "https://oss.sonatype.org/"
   if (isSnapshot.value) Some("snapshots" at nexus + "content/repositories/snapshots")
   else Some("releases" at nexus + "service/local/staging/deploy/maven2")
 }
```
Comment (Tirithel) [inline src/reference/01-General-Info/05-Using-Sonatype.md:187]: Adding information about s01 change for accounts made after Feb 2021.

File: src/reference/01-General-Info/05-Using-Sonatype.md [hunk 3]
```diff
@@ -206,46 +212,59 @@ Close the staging repository and promote the release to central, by hitting
 
 > *Note:* sbt-sonatype is a third-party plugin meaning it is not covered by Lightbend subscription.
 
-To simplify the usage of the Sonatype's Nexus, add the following line to 
+To simplify the usage of the Sonatype's Nexus, add the following line to
 `project/plugins.sbt` to import the [sbt-sonatype plugin][sbt-sonatype] to your project:
 
 ```
-addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "3.9.5")
+addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "3.9.13")
```
Comment (Tirithel) [inline src/reference/01-General-Info/05-Using-Sonatype.md:219]: Update plugin version to get the fix from: 
PR: https://github.com/ShiftLeftSecurity/sbt-ci-release-early/pull/12
Issue: https://github.com/xerial/sbt-sonatype/issues/214

File: src/reference/01-General-Info/05-Using-Sonatype.md [hunk 4]
```diff
@@ -206,46 +212,59 @@ Close the staging repository and promote the release to central, by hitting
...
+
+```scala
+// This becomes a simplified version of the above key.
+publishTo := sonatypePublishToBundle.value
+// Set this to the same value set as your credential files host.
+sonatypeCredentialHost := "oss.sonatype.org"
+// Set this to the repository to publish to using `s01.oss.sonatype.org`
+// for accounts created after Feb. 2021.
+sonatypeRepository := "https://oss.sonatype.org/service/local"
+```
```
Comment (Tirithel) [inline src/reference/01-General-Info/05-Using-Sonatype.md:246]: A bit more information around the newer version of sbt-sonatype.

**Review by eed3si9n — APPROVED** [review]
Thanks, @Tirithel! overall LGTM

**Comment by eed3si9n** [comment]
Probably worth mentioning https://github.com/sbt/sbt-ci-release, which is not always applicable but for any projects using sbt-sonatype it might be a next logical step to just automate the whole thing.

**Comment by Tirithel** [comment]
Good point! I'm just trying to update as I go. I'll check out `sbt-ci-release` when I look into github actions! This does seem like the page to mention it on though :)

**Pull request merged**